### PR TITLE
CF-468: adapt latest changes to Veracode upload script

### PR DESCRIPTION
**Pull request by shintasmith**
To get rid of:
* sleep 5 minutes
* limitation of buildVersion has to be integer

### CODE REVIEW

**Comment by shintasmith** [comment]
@usnavi, @ChandraAddala, this is the same script as the one in cloudfeeds-atomhopper. I just updated the atom-hopper one to be the same.

File: atomhopper/pom.xml [hunk 1]
```diff
@@ -426,39 +426,74 @@
...
-
+    
+                                        // scanName is the version (minus -SNAPSHOT) and build number.
+                                        // an example would be: 1.11.0-1
+                                        def scanName = version - '-SNAPSHOT' + '-' + '${build.number}'
+                                        println "Submitting scan for artifacts with name: " + scanName
+    
+                                        // get the application ID
+                                        // the application must be pre-created from Veracode web UI
                                         def appId =  client.getAppId("Atom Hopper")
```
Comment (ChandraAddala) [inline atomhopper/pom.xml]: Is the app id same for both the projects?

Comment (ChandraAddala) [inline atomhopper/pom.xml]: Nevermind.

**Comment by ChandraAddala** [comment]
Looks good,

File: atomhopper/pom.xml (hunk 1 above)
Comment (shintasmith) [inline atomhopper/pom.xml]: Ya... one is Feeds Atom Hopper, the other is Atom Hopper. :-)

**Pull request merged**